src/operator/every.ts

```diff
@@ -5,6 +5,25 @@ import { Subscriber } from '../Subscriber';
 
 /**
  * Returns an Observable that emits whether or not every item of the source satisfies the condition specified.
+ *
+ * @example <caption>A simple example emitting true if all elements are less than 5, false otherwise</caption>
+ *  Observable.of(1, 2, 3, 4, 5, 6)
+ *     .every(x => x < 5)
+ *     .subscribe(x => console.log(x)); // -> false
+ * 
+ * @example <caption>An example passing this (if you cannot use ES6 arrow function)</caption>
+ * class MyClass {
+ *     private isSmallEnough(n: number): boolean {
+ *         return n < 7;
+ *     }
+ *     constructor() {
+ *         Observable.of(1, 2, 3, 4, 5, 6)
+ *             .every(function(x) { return this.isSmallEnough(x); }, this)
+ *             .subscribe(function(x) { console.log(x); });
+ *     } 
+ * }
+ * let c: MyClass = new MyClass(); // -> true
```

Comment:
The context argument to `every` is a legacy of `Array.prototype.every`. It's not really the important part of the example. Since `isSmallEnough` is a pure function, you can just call it directly. I don't think we need to use the `context` argument in the example.